report_py3o/tests/test_report_py3o.py

```diff
@@ -8,11 +8,17 @@
 import tempfile
 from base64 import b64decode, b64encode
 from contextlib import contextmanager
+from importlib.resources import as_file, files
 from unittest import mock
 
-import pkg_resources
 from PyPDF2 import PdfFileWriter
-from PyPDF2.pdf import PageObject
+
+try:
+    # For PyPDF2 <= 1.26.0
```

Comment:
Is 18.0 still rely on olrder PyPDF2? if not, we should just assume latest version.

Reply:
Well Odoo 18 enforces some minimum python, like I think its 3.10. So it might be that it is actually now impossible to use older PyPDF2 at all. Can you check if thats the case? If so, we can drop this check.